ui/pages/confirmations/components/confirm/footer/footer.tsx

```diff
@@ -155,6 +157,8 @@ const Footer = () => {
       dispatch(
         rejectPendingApproval(currentConfirmation.id, serializeError(error)),
       );
+      dispatch(updateCustomNonce(''));
+      dispatch(setNextNonce(''));
```

Comment:
Not needed now and not introduced by this PR, but perhaps a future refactor could be to remove the nonce from Redux completely and instead just store the custom nonce directly in the `TransactionMeta`, or let the `TransactionController` derive it if not specified.